packages/dashboard-backend/src/constants/schemas.ts

```diff
@@ -53,18 +53,14 @@ export const patchSchema = {
   }]
 };
 
-export const devfileSchema = {
+export const devworkspaceSchema = {
   type: 'object',
   properties: {
-    devfile: {
+    devworkspace: {
       type: 'object'
     }
   },
-  example: {
-    devfile,
-    started: true
-  },
-  required: ['devfile']
+  required: ['devworkspace']
```

Comment:
If you change this object, you should delete 'devfile' from examples too.

https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-backend/src/constants/examples.ts#L71

Reply:
done